src/reactivity/ref.ts

```diff
@@ -117,9 +117,9 @@ export function ref<S, T = unknown, R = HasDefined<S> extends true ? S : RefValu
 ): Ref<R>;
 // implementation
 export function ref(raw?: any): any {
-  // if (isRef(raw)) {
-  //   return {} as any;
-  // }
+  if (isRef(raw)) {
+    return raw;
```

Comment:
We should create a new ref instead of returning `raw`.

Reply:
I disagree. In `vue-next`, we also return the raw value:

https://github.com/vuejs/vue-next/blob/16a737dc6717e1cfe039d4fa56b6eacfc75111c0/packages/reactivity/src/ref.ts#L43-L46